OracleSOASuite/dockerfiles/README.md

```diff
@@ -1,20 +1,190 @@
+
+## Creating a container for AdminServer
+
+Create an environment file adminserver.env.list
+
+    CONNECTION_STRING=<Database Container Name>:<port#>/<ORACLE_PDB>
+    RCUPREFIX=<RCU_Prefix>
+    DB_PASSWORD=<database_sys_password>
+    DB_SCHEMA_PASSWORD=<soa-infra schema password>
+    ADMIN_PASSWORD=<admin_password>
+    MANAGED_SERVER=<managed servername>
+    DOMAIN_TYPE=<soa/osb/bpm>
+Sample Data will look like this...
+
+    CONNECTION_STRING=soadb:1521/soapdb
+    RCUPREFIX=SOA1
+    DB_PASSWORD=Welcome1
+    DB_SCHEMA_PASSWORD=Welcome1
+    ADMIN_PASSWORD=Welcome1
+    MANAGED_SERVER=soa_server1
+    DOMAIN_TYPE=soa
+To start a docker container with a SOA domain and the WebLogic AdminServer call docker run command and pass the above adminserver.env.list file.
+
+A sample docker run command is given below:
+
+     $docker run -i -t  --name soaas --network=SOANet -p 7001:7001  -v /scratch/DockerVolume/SOAVolume/SOA:/u01/oracle/user_projects   --env-file ./adminserver.env.list oracle/soa:12.2.1.3
+The options "-i -t" in the above command runs the container in interactive mode and you will be able to see the commands running in the container. 
+This includes the command for RCU creation, domain creation and configuration followed by starting the Admin Server. 
+Mapping container port 7001 to host port 7001 enables accessing of the Weblogic host outside of the local host.
+Connecting to SOANet network enables accessing the DB container by it's name i.e soadb.
+This includes the command to tail logs to keep the container up and running.
+
+IMPORTANT: You need to wait till all the above commands are run before you can access the AdminServer Web Console.
+
+## Creating a container for SOA Server
+
+Create an environment file soaserver.env.list with the below details:
+
+    MANAGED_SERVER=<managed server name, For Exp:- soa_server1, osb_server1, bpm_server1>
+    DOMAIN_TYPE=<soa/osb/bpm>
+    ADMIN_HOST=<Admin host name>
+    ADMIN_PORT=<port number where Admin Server is running>
+Sample Data will look like this
+
+    MANAGED_SERVER=soa_server1
+    DOMAIN_TYPE=soa
+    ADMIN_HOST=slc09cwi.us.oracle.com
+    ADMIN_PORT=7001
+To start a docker container for SOA server you can simply call docker run command and passing soaserver.env.list. 
+
+A sample docker run command is given below:
+
+    $ docker run -i -t  --name soams --network=SOANet -p 8001:8001   --volumes-from soaas   --env-file ./soaserver.env.list oracle/soa:12.2.1.3 "/u01/oracle/dockertools/startMS.sh"
+    
+Using --volumes-from reuses the volume created by the Admin container.
+This includes the command to tail logs to keep the container up and running.
+
+IMPORTANT: You need to wait till all the above commands are run before you can start Admin and SOA Servers.
+
+Once the SOA container is created logs will be tailed and displayed to keep the container running.
+
+Now you can access the AdminServer Web Console at http://:7001/console  with weblogic/Welcome1 credentials.
```

Comment:
Hostname missing from the URL.

Reply:
Also, I would combine all four of the URL lines into a single bullet list instead.